[ ERROR ]  runJest: TypeError: runCLI is not a function - with Jest 25

<!--
NOTE:
Before submitting an issue, please consult our docs -> https://stenciljs.com/
-->

**Stencil version:**
<!-- (run `npm list @stencil/core` from a terminal/cmd prompt and paste output below): -->
```
 @stencil/core@1.9.0-14
```

**I'm submitting a:**
<!-- (check one with "x") -->
[x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://stencil-worldwide.herokuapp.com/ or https://forum.ionicframework.com/

**Current behavior:**
<!-- Describe how the bug manifests. -->

```
C:\ae\adaept.com\_components_\ae-icon5-component>npm test

> @adaept/ae-icon5-component@0.9.996 test C:\ae\adaept.com\_components_\ae-icon5-component
> stencil test --spec --e2e

[26:03.0]  @stencil/core v1.9.0-14
[26:03.1]  testing e2e and spec files
[26:03.1]  build, aeicon5component, dev mode, started ...
[26:03.9]  transpile started ...
[26:06.2]  transpile finished in 2.28 s
[26:06.3]  copy started ...
[26:06.6]  generate styles started ...
[26:06.6]  bundling components started ...
[26:07.0]  generate styles finished in 448 ms
[26:09.0]  bundling components finished in 2.36 s
[26:12.3]  copy finished (2460 files) in 6.03 s
[26:12.4]  build finished in 9.30 s

[26:12.6]  jest args: --e2e --spec --max-workers=4

[ ERROR ]  runJest: TypeError: runCLI is not a function


npm ERR! Test failed.  See above for more details.

C:\ae\adaept.com\_components_\ae-icon5-component>
```
**Expected behavior:**
<!-- Describe what the behavior would be without the bug. -->

Tests work

**Steps to reproduce:**
<!-- If you are able to illustrate the bug or feature request with an example, please provide steps to reproduce and if possible a demo
-->

Repo: https://github.com/adaept/ae-icon5-component


**Related code:**

```tsx
// insert any relevant code here
```

**Other information:**
<!-- List any other information that is relevant to your issue. Stack traces, related issues, suggestions on how to fix, Stack Overflow links, forum links, etc. -->

--- a/src/testing/jest/jest-runner.ts
+++ b/src/testing/jest/jest-runner.ts
@@ -28,7 +28,7 @@ export async function runJest(config: d.Config, env: d.E2EProcessEnv) {
 
     // run the jest-cli with our data rather than letting the
     // jest-cli parse the args itself
-    const { runCLI } = require('jest-cli');
+    const { runCLI } = require('@jest/core');
     const cliResults = await runCLI(jestArgv, projects);
 
     success = !!cliResults.results.success;

